Source/CardUtils.swift

```diff
@@ -1,65 +1,19 @@
 import Foundation
+import Checkout
 
 /// Payment Form Utilities
 public class CardUtils {
```

Comment:
Do we need this ? I am expecting this should be inside checkout iOS SDK

Reply:
card gaps and formatting are currently not part of the checkout sdk. we can move them over in future tickets

Reply:
Task : https://checkout.atlassian.net/browse/PIMOB-1193